exo-go/src/template_helpers/index.go

```diff
@@ -32,6 +33,8 @@ dependencies:
 services:
   public:
   private:
+
+templates:
```

Comment:
The problem is, simply overriding the Git remote of the template causes a ton of edge cases that we rather not deal with. For example, we have to delete the files and clone them. There is Git history stored on the local client (in the `.git` folder). If we start changing the remote of the Git repo to another repository that contains completely different files and history, bad things will happen, and it might not even be possible to pull or push correctly. 

I think it's safer to abort with an error if the template to be added already exists, and force the user to delete the template first and then set it up cleanly with a new URL. 
That's how `git remote` does it too.